client/src/components/application/Examine/CompNameIcon.vue

```diff
@@ -2,7 +2,6 @@
 <template>
   <v-icon v-if="state"
           :style="setup.style"
-          class="fw-700"
```

Comment:
Un-bolded these icons globally -- so the edit page looks like the transactions page.